pkg/logs/query.go

```diff
@@ -72,36 +72,36 @@ func Query(
 
 	req, err := http.NewRequest(http.MethodGet, endpoint.String(), nil)
 	if err != nil {
-		return warn, errors.Wrap(err, "creating request")
+		return http.StatusUnprocessableEntity, warn, errors.Wrap(err, "creating request")
```

Comment:
I think here we could improve the error msg too - maybe something like `could not create request`

Reply:
👍